packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -127,19 +116,16 @@ Each value for `response_mode` delivers different behavior:
 * `okta_post_message` - Uses [HTML5 Web Messaging](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) (for example, window.postMessage()) instead of the redirect for the authorization response from the `/authorize` endpoint.
 
     `okta_post_message` is an adaptation of the [Web Message Response Mode](https://tools.ietf.org/html/draft-sakimura-oauth-wmrm-00#section-4.1).
-    This value provides a secure way for a single-page application to perform a sign-in flow
-    in a popup window or an iFrame and receive the ID token and/or access token back in the parent page without leaving the context of that page.
-    The data model for the postMessage call is in the next section.
-* The `Referrer-Policy` header is automatically included in the request for `fragment` or `query`, and is set to `Referrer-Policy: no-referrer`.
+    This value provides a secure way for a single-page application to perform a sign-in flow in a pop-up window or an iFrame and receive the ID token and/or access token back in the parent page without leaving the context of that page. The data model for the postMessage call is in the next section.
```

Comment:
Not in the scope of your change, but this can now be a `code` response if you feel like cleaning it up.